sysbench/oltp_insert_ignore.lua

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env sysbench --config-file=config
+
+-----------------------------------
+-- Insert Ignore OLTP benchmark
+----------------------------------
+
+require("oltp_common")
+
+function create_table(drv, con, table_num)
+   print(string.format("Creating table 'sbtest%d'...", table_num))
+   local query = string.format([[
+CREATE TABLE sbtest%d(
+  id INTEGER NOT NULL,
+  k INTEGER DEFAULT '0' NOT NULL,
+  c CHAR(120) DEFAULT '' NOT NULL,
+  pad CHAR(60) DEFAULT '' NOT NULL,
+  PRIMARY KEY (id)
+)]],
+      table_num)
+
+   con:query(query)
+
+   print(string.format("Creating a secondary index on 'sbtest%d'...",
+                       table_num))
+   con:query(string.format("CREATE UNIQUE INDEX k_%d ON sbtest%d(k)",
+                           table_num, table_num))
+end
+
+function prepare_statements()
+end
+
+function event()
+  local table_name = "sbtest" .. sysbench.rand.uniform(1, sysbench.opt.tables)
+  local query = string.format("INSERT IGNORE INTO %s (id, k, c, pad) VALUES ", table_name)
+  for i = 1, 5 do
```

Comment:
should we use a larger number, I think 5 is a bit small.

Reply:
This is not a prepared statement, if we use a lot of VALUES, the performance would be affected by the parser (the dominant time cost) and then the test can't reflect the truth.

5 is enough to cover insert ignore code and prevent the performance regression if a request visit TiKV multiple times.

Reply:
It is OK. Below 100 values, the effects of parser should be negligible. Also, you could also use the prepare statement.

Reply:
> Also, you could also use the prepare statement.

emmmm... I'd like to but I'm not so familiar with Lua this bench framework... so just get things done

Reply:
How about adding a parameter: Eg, https://github.com/pingcap/productivity-tools/blob/master/benchmark/batch_insert/batch_insert.lua#L65